x/stakeibc/keeper/msg_server_claim_undelegated_tokens.go

```diff
@@ -68,10 +71,23 @@ func (k Keeper) GetClaimableRedemptionRecord(ctx sdk.Context, msg *types.MsgClai
 	}
 
 	// check that the record is claimable
-	if !userRedemptionRecord.GetIsClaimable() {
-		errMsg := fmt.Sprintf("User redemption record %s is not claimable", userRedemptionRecord.Id)
+	hostZoneUnbonding, found := k.RecordsKeeper.GetHostZoneUnbondingByChainId(ctx, userRedemptionRecord.EpochNumber, msg.HostZoneId)
+	if !found {
+		errMsg := fmt.Sprintf("Host zone unbonding record %s not found on host zone %s", userRedemptionRecordKey, msg.HostZoneId)
+		k.Logger(ctx).Error(errMsg)
+		return nil, sdkerrors.Wrapf(types.ErrInvalidUserRedemptionRecord, "could not get host zone unbonding record: %s", userRedemptionRecordKey)
```

Comment:
sorry to be very nit, but could we wrap this in fmt.Sprintf ? Another easy option is to just return errMsg

Reply:
done